src/content/reference/react-dom/components/input.md

```diff
@@ -32,6 +32,8 @@ To display an input, render the [built-in browser `<input>`](https://developer.m
 
 `<input>` supports all [common element props.](/reference/react-dom/components/common#props)
 
+* [`formAction`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input#formaction): A string or function. Overrides the parent `<form action>` for `type="submit"` and `type="image"`. When a URL is passed to `action` the form will behave like a standard HTML form. When a function is passed to `formAction` the function will handle the form submission. See [`<form action>`](/reference/react-dom/components/form#props).
```

Comment:
Should we put this as a separate point (like not a sub-point under "supports all common element props" and call out that this is a React extension and only available for canary?

Reply:
Do you think we need a callout here in addition to the page-level canary callout at the top of the page?

Reply:
Good catch, fixed.